### PR TITLE
feat(tray): add `TrayIcon::rect` method

### PR DESCRIPTION
**Pull request by pronebird**
Follow up PR on https://github.com/tauri-apps/tray-icon/pull/147

### CODE REVIEW

File: core/tauri/src/tray/mod.rs [hunk 1]
```diff
@@ -420,6 +420,18 @@ impl<R: Runtime> TrayIcon<R> {
...
+  pub fn rect(&self) -> crate::Result<Option<crate::Rect>> {
+    run_item_main_thread!(self, |self_: Self| self_.inner.rect().map(
+      |frame| {
+        Rect {
+          position: frame.position.into(),
+          size: frame.size.into(),
+        }
+      }
+    ))
+  }
```
Comment (amrbashir) [inline core/tauri/src/tray/mod.rs]:
```suggestion
  
  /// Get tray icon rect.
  ///
  /// ## Platform-specific:
  ///
  /// - **Linux**: Unsupported, always returns `None`.
  pub fn rect(&self) -> crate::Result<Option<crate::Rect>> {
    run_item_main_thread!(self, |self_: Self| self_.inner.rect().map(
      |rect| {
        Rect {
          position: rect.position.into(),
          size: rect.size.into(),
        }
      }
    ))
  }
```

**Review by amrbashir — CHANGES_REQUESTED** [review]
Thank you, Could you also add a change file in `.changes` directory?

**Comment by pronebird** [comment]
> Thank you, Could you also add a change file in `.changes` directory?

In a minute

**Comment by pronebird** [comment]
Not sure how exactly you name `.md` files under `changes/`. Please have another look.

File: .changes/tray-icon-rect.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"tauri": "feat"
```
Comment (amrbashir) [inline .changes/tray-icon-rect.md]:
```suggestion
"tauri": "minor:feat"
```

**Comment by amrbashir** [comment]
Once a new release of tray-icon crate is made, we can merge this

**Review by amrbashir — DISMISSED** [review]

**Review by amrbashir — APPROVED** [review]

**Comment by amrbashir** [comment]
Thank you

**Pull request merged**